packages/extension/src/routes/Home.vue

```diff
@@ -210,12 +221,39 @@ export default {
       this.$refs.context.open(event, post);
     },
 
+    onPostDndMenu(event) {
+      ga('send', 'event', 'Dnd', 'Menu');
+      this.$refs.dndContext.open(event);
+    },
+
     onPostMenuOpened(event, post) {
       const rect = event.target.getBoundingClientRect();
       this.$refs.context.positionMenu({ bottom: rect.top - 8, right: rect.right });
       this.selectedPostId = post.id;
     },
 
+    onDndMenuOpened(event) {
+      const rect = event.event.target.getBoundingClientRect();
+      this.$refs.dndContext.positionMenu({ top: rect.bottom + 7, right: rect.right });
```

Comment:
We use `8px` gap usually. InVision sometime messes the numbers :flushed: 

```suggestion
      this.$refs.dndContext.positionMenu({ top: rect.bottom + 8, right: rect.right });
```